src/www/ui/api/Controllers/LicenseController.php

```diff
@@ -366,4 +367,31 @@ private function isNewLicense($shortName, $groupId = 0)
       $statement);
     return $result["cnt"] == 0;
   }
+
+  /**
+   * Delete license candidate by id.
+   *
+   * @param Request $request
+   * @param ResponseHelper $response
+   * @param array $args
+   * @return ResponseHelper
+   */
+  public function deleteAdminLicenseCandidate($request, $response, $args)
+  {
+    $resInfo = null;
+    if (!Auth::isAdmin()) {
+      $resInfo = new Info(400, "Only admin can perform this operation.", InfoType::ERROR);
+    } else {
+      $id = intval($args['id']);
+      $adminLicenseCandidate = $this->restHelper->getPlugin('admin_license_candidate');
+
+      if ($adminLicenseCandidate->getDataRow($id)) {
+        $res = $adminLicenseCandidate->doDeleteCandidate($id);
+        $resInfo = new Info($res->getStatusCode(),"License candidate will be deleted.", $res->getStatusCode() == 200 ? InfoType::INFO : InfoType::ERROR);
```

Comment:
Please read the message if the license candidate cannot be deleted.

Reply:
Done 👍